### PR TITLE
YLP-476 Team: Modal add new HCP member

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/pages/hcp/team-member-add-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+function AddMemberDialog(props: AddMemberDialogProps): JSX.Element | null {
+  const { addMember } = props;
+
+  const [email, setEMail] = React.useState("");
+  const [role, setRole] = React.useState("viewer" as TeamMemberRole);
+  const [buttonDisabled, setButtonDisabled] = React.useState(true);
+  const classes = dialogClasses();
+
+  let teamName = "";
```
Comment (leovalette) [inline packages/yourloops/pages/hcp/team-member-add-dialog.tsx:73]:
```suggestion
  const teamName = addMember?.team?.name  ?? "";
```

File: packages/yourloops/pages/hcp/team-member-add-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    teamName = addMember.team.name;
+  }
+
+  const handleChangeEMail = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>): void => {
+    const eMail = e.target.value;
+    setEMail(eMail);
+    setButtonDisabled(!REGEX_EMAIL.test(eMail));
+  };
+  const handleChangeRole = (_e: React.ChangeEvent<HTMLInputElement>, checked: boolean): void => {
+    setRole(checked ? "admin" : "viewer");
```
Comment (leovalette) [inline packages/yourloops/pages/hcp/team-member-add-dialog.tsx:84]: Maybe we should have an enum for the different roles ?

Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/team-member-add-dialog.tsx:84]: Yes good idea.
They have now implemented it, I didn't check the new stuff available in the language.

File: packages/yourloops/pages/hcp/team-member-add-dialog.tsx (hunk 1 above)
Comment (yvon-dblg) [reply packages/yourloops/pages/hcp/team-member-add-dialog.tsx:73]: Yes, a WIP code left in bad shape, sorry.

**Pull request merged**